pkg/monitoring/virt-controller/metrics/vm_metrics.go

```diff
@@ -0,0 +1,24 @@
+package metrics
+
+import (
+	"github.com/machadovilaca/operator-observability/pkg/operatormetrics"
+	v1 "kubevirt.io/api/core/v1"
+)
+
+var (
+	vmMetrics = []operatormetrics.Metric{
+		vmsCreatedCounter,
+	}
+
+	vmsCreatedCounter = operatormetrics.NewCounterVec(
+		operatormetrics.MetricOpts{
+			Name: "kubevirt_vm_created_total",
+			Help: "Amount of VMs created, broken down by namespace.",
+		},
+		[]string{"namespace"},
+	)
+)
+
+func NewVMCreated(vm *v1.VirtualMachine) {
```

Comment:
Can we reduce the `vm` to `namespace`?

Reply:
since we are using a pointer I think we don't have any performance problems with copying the structure, and personally, I think it makes more sense that a function `NewVMCreated` receives a VM as an argument and not a Namespace. Even if in the future we want more labels or create new metrics in the function, it would be easier